fms/utils/serialization.py

```diff
@@ -446,6 +456,7 @@ def _load_partial_state_dict(
     model: torch.nn.Module,
     state_dict,
     needs_tp_sharding: bool,
+    linear_type: str,
```

Comment:
this doesn't seem necessary
linear_type is a property of the individual modules
Also we should consider the possibility of having mix linear types in one model